src/main/resources/db/migration/V4__ny_datamodell.sql

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS dialog;
+
+CREATE TABLE dialog
+(
+    id            UUID UNIQUE NOT NULL PRIMARY KEY,
+    sykmelding_id UUID UNIQUE NOT NULL,
```

Comment:
Vi snakket vel så vidt om et mulig behov for å sende løse dialoger, uten en sykmelding. 
Hvordan skal i så fall det løses, hvis sykmelding_id må være satt?